test/interop/python/noLibFlag/checkDiffPythonModname.prediff

```diff
@@ -3,6 +3,7 @@
 # Remove clang/gcc unused argument warning
 cat $2 | sed '/^clang: warning: argument unused/d' | \
 sed '/warning: unrecognized command line option/d' | \
+sed '/warning: code will never be executed/d' | \
```

Comment:
Looks like these .prediff files could also get the symbolic link treatment, no? either way I appreciate the progress here & you can merge the PR without doing it

Reply:
These file-specific prediffs are also invoking the python module with a specific name, whereas the global prediff looks like it just does filtering.